10-validation.md

```diff
@@ -1,79 +1,77 @@
-# End-to-End Validation
+# End-to-End Multi Cluster Validation
 
-Now that you have a workload deployed, and the [Azure FrontDoor instance that will distribute the traffic between your clusters in different regions](./11-frontdoor.md), you can start validating and exploring this reference implementation of the [AKS secure baseline cluster](./). In addition to the workload, there are some observability validation you can perform as well.
+Now that you have a workload deployed, the [ASP.NET Core Docker sample web app](./09-workload.md), you can start validating and exploring this reference implementation of the [AKS multi cluster](./). In addition to the workload, there are some observability validation you can perform as well.
 
-## Validate the Web App
+## Validate the Region Failover
 
-This section will help you to validate the workload is exposed correctly and responding to HTTP requests.
+This section will help you to validate the workload is exposed correctly and responding to HTTP requests. Additionally, the workload is now responding from multiple regions, and you want to test the failover.
 
 ### Steps
 
+1. Get your Azure Application Gateway instance names
+
+   ```bash
+   APPGW_FQDN_BU0001A0042_03=$(az deployment group show -g rg-bu0001a0042-03 -n cluster-stamp --query properties.outputs.agwName.value -o tsv)
+   APPGW_FQDN_BU0001A0042_04=$(az deployment group show -g rg-bu0001a0042-04 -n cluster-stamp --query properties.outputs.agwName.value -o tsv)
+   ```
+
 1. Get your Azure Front Door public DNS name
 
-   > :book: The app team conducts a final acceptance test to be sure that traffic is flowing end-to-end as expected, so they place a request against the Azure Front Door endpoint.
+   > :book: the app team conducts a final system test to be sure that traffic is flowing end-to-end as expected, so they are about to place some requests against the Azure Front Door endpoint.
 
    ```bash
    # query the Azure Front Door FQDN
    FRONTDOOR_FQDN=$(az deployment group show -g rg-bu0001a0042-shared -n shared-svcs-stamp --query properties.outputs.fqdn.value -o tsv)
    ```
 
-1. Browse to the site.
+1. Add some load to the web application to test it consistently respond `HTTP 200`, even though it is wrestling beahind the scenes with two different region outage incidents.
+
+   > :book: the app team configured Azure Front Door backend pool to balance the traffic equally between the two regions. It will do a round robing between them as long as the roundtrip lantency from their clients POP(s), and the two regions are the same. Otherwise, the client traffic is going to flow to a single direction which is the closest region based on the sampled latency.
```

Comment:
```suggestion
   > :book: The app team configured Azure Front Door backend pool to balance the traffic equally between the two regions. It will do a round robing between them as long as the roundtrip latency from their clients POP(s), and the two regions are the same. Otherwise, the client traffic is going to flow to a single destination which is the closest region based on the sampled latency.
```